src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+for (const row of rows) {
+  const secondColumnCopy = row.children[1].cloneNode(true);
+
+  row.insertBefore(secondColumnCopy, row.children[4]);
```

Comment:
What is 1 or 4 ?
Let's move these numbers to specific variables, so we could change their values later

Reply:
Thank you, I moved them to variables